### ISSUE TITLE
About the pre-trained models

### ISSUE THREAD

**Issue by jrhendrix** [nanoporetech/remora#32]
I am wondering how the pre-trained models were made, specifically what organism did the training data come from?

As I understand, basecalling accuracy for a sample can be improved by using a model trained on data coming from the same taxonomy as the sample. Does the accuracy of modification-calling also benefit from taxon-specific training data?

**Comment by marcus1487** [comment]
The latest tech update from Clive and the presentations at London calling describe the training samples. Most current Remora models are trained from PCR human data treated with the M.SssI bacterial methylase. The 5hmC models are treated further to convert 5mC to 5hmC. The all-context model is trained from printed oligo randomers and we are hoping to move to models trained completely from non-genomic randomers soon. 

Remora only takes 100 samples as input, so there is unlikely to be much advantage to training a Remora model for a specific organism when the modeled sequence context is the same. Most released models only run in CG-context, so organisms with modified bases outside of CG contexts would need a model trained to the new contexts. But I would suspect that these would not need data from an organism of interest, but only from a motif of interest given the small window of input signal. This is being tested internally using the randomer training data.

**Comment by benbfly** [comment]
Hi @marcus1487 ,
Have you and ONT published the protocol of how the randomers are being synthesized (down to the level of oligo vendor, etc)?  If not, do you know if this is something that will be published to allow the community to try this with their modifications of interest?

**Comment by cjw85** [comment]
Hi @benbfly,

We cannot provide further details at this time.
